src/app/project/board/project-tab-view/project-tab-view.component.spec.ts

```diff
@@ -0,0 +1,112 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { ProjectTabViewComponent } from './project-tab-view.component';
+import { Component, ViewChild } from '@angular/core';
+
+/**
+ * Test host component to simulate parent component.
+ */
+@Component({
+    selector: 'app-board-host-component',
+    template: '<app-project-tab-view [metadata]="projectMetadata"></app-project-tab-view>'
+})
+class TestHostBoardComponent {
+
+    @ViewChild('projectTabView') projectTabView: ProjectTabViewComponent;
+
+    // input parameters
+    projectMetadata =  {
+        'alternateName': 'test',
+        'contactPoint': {
+            'address': {'addressLocality': 'Basel', 'postalCode': '4000', 'streetAddress': 'Teststrasse'},
+            'email': 'stewart.abraham@test.ch',
+            'familyName': 'Abraham',
+            'givenName': 'Stewart',
+            'jobTitle': 'Dr.',
+            'memberOf': 'http://ns.dasch.swiss/test-dasch',
+            'sameAs': {'type': 'https://schema.org/URL', 'value': 'https://orcid.org/0000-0002-1825-0097'},
```

Comment:
what does `sameAs` mean? Is it possible to have a more explicit label for this field?

Reply:
From my understanding, `sameAs` contains the profile url. It is the key I received in REST API response. I will add this point in metadata discussion if we want to change it. I am adding @BalduinLandolt and @mpro7 for reference.

Reply:
I guess you can find a better label. I really doubt that a lambda user will understand what `sameAs` means. You can also discuss it with Marion or Rita.